Multimux, tribuffer, better icons, and inout support

nobodywasishere opened the pull request:
- Modifies most current icons to make them more even and work better with thicker line strokes
- Adds a multimux if bus width input is greater than 1
- Line endings are now rounded
- Adds tribuffer icon
- Fixes #82: adds `"inout"` and `"z"` to `lib/yosys.schema.json5`, which are treated like `"output"` and `"x"`, respectively

## Example 1

Before:
![Screenshot_2021-03-20_01-28-06](https://user-images.githubusercontent.com/32797552/111860091-8d23fb80-891b-11eb-98f5-428a8eb40425.png)

After:
![Screenshot_2021-03-20_01-27-02](https://user-images.githubusercontent.com/32797552/111860075-6c5ba600-891b-11eb-9fe9-472f78f1f396.png)

## Example 2

Before:
![Screenshot_2021-03-20_01-16-02](https://user-images.githubusercontent.com/32797552/111860127-ca888900-891b-11eb-8d4b-4c375e214eb3.png)

After:
![Screenshot_2021-03-20_01-29-37](https://user-images.githubusercontent.com/32797552/111860128-cf4d3d00-891b-11eb-8f99-d700f7eee913.png)

nturley commented:
Thanks for the PR. This is excellent. My only problem is that it hardcodes in the list of mux aliases. Rather than hardcoded substitution just for MUXes, we should allow any symbol in the skin file to have single bit and multibit variants.

nobodywasishere commented:
Since I'm using Yosys parameters to determine whether it's a single or multibit variant of the mux, this does involve #8. 

Based on [this](https://github.com/YosysHQ/yosys/blob/master/manual/CHAPTER_CellLib.tex), it appears only muxes (and their variants), tribuffers, dff's, sr latches (not in digital skin file), and memrd/memwd/meminit/mem (not in digital skin file) have a `WIDTH` parameter. The others (add, sub, and, etc.) have individual `A_WIDTH` and `B_WIDTH`, etc which don't fit quite as well with the idea of multibit variants, at least to me. I could be wrong though.

I've updated it to what I think is a better method of detection / selecting multibit variants, where it will just append `-bus` to the end of the type instead of replacing the type altogether. While there is only mux multibit variant, others can be added fairly easily now in the skin file just by moving around the aliases (for dff, sr, and tribuf, or any that I missed that have a `WIDTH` parameter).

I added an alias to the generic type just in case it happens to have a `WIDTH` parameter.

nobodywasishere commented:
#103 is to fix a mistake I created in this PR. The generic type alias currently breaks the names overtop generics so they'll show up as i.e. `\13` instead of the proper cell name. Apologies for this.
